src/components/dialogmote/referat/Deltakere.tsx

```diff
@@ -96,38 +93,34 @@ const DeltakerBehandler = ({ behandler }: DeltakerBehandlerProps) => {
     <DeltakerEkspanderbartPanel
       ariaLabel={tittelTekst}
       tittel={
-        <FlexRow>
+        <div className="flex gap-2">
           <img
             src={MedisinskrinImage}
             alt="Medisinskrin-ikon"
             {...deltakerIconProps}
           />
-          <DeltakerTekst size="small">{tittelTekst}</DeltakerTekst>
-        </FlexRow>
+          <Heading size="small">{tittelTekst}</Heading>
+        </div>
       }
     >
-      <FlexRow topPadding={PaddingSize.SM}>{texts.behandlerTekst}</FlexRow>
-      <FlexRow topPadding={PaddingSize.MD}>
+      <div className="flex flex-col gap-4 mt-4 mb-4">
+        <p>{texts.behandlerTekst}</p>
```

Comment:
Noen Aksel-komponenter man kunne brukt i stedet for `<p>`? `BodyLong` eller noe sånt?